pkg/cnab/provider/testdata/bundle-with-param-sources.json

```diff
@@ -99,7 +99,8 @@
   },
   "definitions": {
     "connstr": {
-      "type": "string"
+      "type": "string",
```

Comment:
Now that you validate default values, the test started failing because this didn't have a default.